src/seedu/addressbook/data/AddressBook.java

```diff
@@ -107,6 +107,14 @@ public void clear() {
         allTags.clear();
     }
 
+    /**
+     * Sorts address book entries alphanumerically
+     * Returns a new alphanumeric sorted UniquePersonList of all persons in the address book at the time of the call.
```

Comment:
Since this method does not returns a UniquePersonList, perhaps `Sorts address book entries alphanumerically` is fine =)

Reply:
Thank you!